src/test/java/page/NewContactsModal.java

```diff
@@ -0,0 +1,67 @@
+package page;
+
+import elements.DropDown;
+import elements.Input;
+import elements.Search;
+import elements.TextArea;
+import objects.Contacts;
+import org.openqa.selenium.By;
+import org.openqa.selenium.TimeoutException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+
+public class NewContactsModal extends BasePage {
+    public NewContactsModal(WebDriver driver) {
+        super(driver);
+    }
+
+    By SELECT = By.cssSelector("[class~=select]");
+    By SAVE_BUTTON = By.xpath("//*[@class='button-container-inner slds-float_right']/button[3]");
```

Comment:
this locator should be written in such way:
//button[@title='Save']

Reply:
fix